projects/habitat_ovmm/configs/agent/floorplanner_eval.yaml

```diff
@@ -64,7 +64,7 @@ AGENT:
       type: modular
 
     PLACE:
-      type: hardcoded
+      type: modular_debug # or "hardcoded"
```

Comment:
imo, the whole hardcoded place logic should be deleted.

Reply:
i do think instead of "modular" we need to call this "heuristic" - everything we're doing is modular

Reply:
even the learned one

Reply:
Agreed. Renamed the PR and will also update the code accordingly.

Reply:
Please keep the hardcoded one for now. It is giving some cases where the place policy falsely succeeds. We can remove it later.